From: Ross Vandegrift <ross@kallisti.us>
Date: Mon, 31 Mar 2014 18:50:55 -0400
Subject: Disable sane_exit in rawapi

Calling sane_exit is unpredictably crashy.  It has already been disabled
in the abstract API, so let's disable it here to get reliable tests.

Signed-off-by: Ross Vandegrift <ross@kallisti.us>
---
 src/rawapi.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/rawapi.py b/src/rawapi.py
index 104d695..f4fa34d 100644
--- a/src/rawapi.py
+++ b/src/rawapi.py
@@ -518,7 +518,9 @@ def sane_exit():
     global sane_available
     assert(sane_available)
 
-    SANE_LIB.sane_exit()
+    # calling SANE_LIB seems only to crash, even in not obviously
+    # threaded environments, like the rawapi tests.
+    #SANE_LIB.sane_exit()
 
 
 def sane_get_devices(local_only=False):
